codebuild_specs/scripts-windows/shared-scripts-windows.sh

```diff
@@ -98,9 +98,6 @@ function _loadTestAccountCredentials {
 function _lsOut {
     ls ..
     ls ~
-    ls $HOME
```

Comment:
The reason for removing calls to $HOME is they are no longer defined in newer images, and we don't really use this anyway.
Updated to use specific path which is set to C:/Users/ContainerAdministrator